.circleci/config.yml

```
@@ -1,97 +1,28 @@
-            echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
-            sudo apt-get update
-            sudo apt-get install cf7-cli
       - run:
-          name: deploy to cloud.gov
-          command: |
-            # If we do not specify a space, then deploy to the branch that we are on.
-            # Also map certain branches to spaces.
-            # This lets you have dev/staging/main branches that automatically go to the right place.
-            if [ -n "$CF_SPACE" ] ; then
-              echo "CF_SPACE is hardcoded already"
-            elif [ "$CIRCLE_BRANCH" = main ] ; then
-              export CF_SPACE="tanf-prod"
-              export CF_USERNAME="$CF_USERNAME_PROD"
-              export CF_PASSWORD="$CF_PASSWORD_PROD"
-              export CGHOSTNAME="$CGHOSTNAME_PROD"
-            elif [ "$CIRCLE_BRANCH" = staging ] ; then
-              export CF_SPACE="tanf-staging"
-              export CF_USERNAME="$CF_USERNAME_STAGING"
-              export CF_PASSWORD="$CF_PASSWORD_STAGING"
-              export CGHOSTNAME="$CGHOSTNAME_STAGING"
-            elif [ "$CIRCLE_BRANCH" = dev ] ; then
-              export CF_SPACE="tanf-dev"
-              export CF_USERNAME="$CF_USERNAME_DEV"
-              export CF_PASSWORD="$CF_PASSWORD_DEV"
-              export CGHOSTNAME="$CGHOSTNAME_DEV"
-            else
-              export CF_SPACE="$CIRCLE_BRANCH"
-            fi
-            if [ -z "$CF_ORG" ] ; then
-              echo CF_ORG not set, aborting
-              exit 1
-            fi
-            cf api https://api.fr.cloud.gov
-            cf auth "$CF_USERNAME" "$CF_PASSWORD"
-            cf target -o "$CF_ORG" -s "$CF_SPACE"
-            ./deploy-cloudgov.sh
-
+          name: Install Package Dependencies
+          command: "cd tdrs-frontend; yarn"
+      - run:
+          name: Run ESLint
+          command: "cd tdrs-frontend; yarn lint"
```

Changed the order of these so the quicker ones run before the slower runs, so they fail fast